### ISSUE TITLE
Random bot focus in player active zones

### ISSUE THREAD

**Issue by Vortikai** [liyunfan1223/mod-playerbots#765]
**Is your feature request related to a problem? Please describe.**

The level settings are a nice good start, but I'm frustrated there's 100+ max level bots that aren't being seen on the server when the actual player(s) on the server are 1-20, with about 10 random bots that are levels 1-10 or 12 random bots levels 10-20.

**Describe the solution you'd like**

I would like a system that populates the server with random bots at random levels, but essentially initialize random bot levels near the max level of the actual players, and make more bots active in the zone the players are in. This is a player-focused approach

A simplistic example is 10 random bots level 1-10 in each starting zone, 10 random bots level 20-30 in appropriate zones, 10 level 30-40 in appropriate zones, etc.

A new player creates an undead rogue and joins the world starting at level 1.

The system initializes 40-50 random bots, level 1 (or 1-10) and teleports them to Tirisfal Glades.

The undead rogue logs out and logs on a troll hunter, level 64.

The system inits 40-50 random bots, level 60-70, and teleports them to the zone the troll hunter is in.

**Describe alternatives you've considered**

Random bot level curves around player level, similar to sync level with players option.

Another alternative I've considered is a random bot level curve for the entire server. Right now I have 40 bots levels 1-10, but over 160 levels 70-80. I'd like to flip that around so that the server is more low-level friendly.

**Additional context**

I'm new and there might already be a way to do this I'm unaware of. Right now the only solution I can make the zones feel more alive is to snyc max level with player and .playerbots rndbot reset/.playerbots rndbot init with a server restart. As I gain levels, the fewer bots I will come across though.

**Comment by hermensbas** [comment]
It still needs more love, once i got more time i will get back at it.

**Comment by Dreathean** [comment]
I know it is not quite like the automatically adjusting system you're suggesting here, but just wanted to throw this is in case you haven't seen it already as it should help do close to what you're saying otherwise - https://github.com/DustinHendrickson/mod-player-bot-level-brackets

**Comment by Vortikai** [comment]
Yeah I'm using that. It's not quite the same as cities are still pretty empty and such but it helps a little
